integration_test/src/python/topology_test_runner/resources/test.json

```diff
@@ -4,49 +4,56 @@
   "role" : "dummy_role",
   "env" : "dummy_env",
   "cliConfigPath" : "$HOME/.heron/conf",
-  "processingType" : "non_stateful",
+  "processingType" : "stateful",
```

Comment:
With the changes to this file, the normal topologies aren't tested?

Reply:
No. This will only enable instance state check in stateful topologies, but won't disable DAG structure check for both normal and stateful topologies.